microsoft-edge/webview2/reference/javascript/sharedbufferreceivedevent.yml

```
@@ -0,0 +1,131 @@
+### YamlMime:TSType
+name: SharedBufferReceivedEvent
+uid: 'WebView2Script!SharedBufferReceivedEvent:class'
+package: WebView2Script!
+fullName: SharedBufferReceivedEvent
+summary: >-
+  Event object for the `chrome.webview.sharedbufferreceived` event. This event is dispatched when
+  `CoreWebView2.PostSharedBufferToScript` is successfully called.
+remarks: |-
+
+
+  #### Examples
+
+  For Example, if we want to send data to script for one time read only consumption,
+
+  In native host app code, set data into the shared memory.

```

```suggestion
First, in the native host app code, set data into the shared memory:
```
Seems like this needs smoother wording than: 
set data into the shared memory
What does that mean?  set data into??  Does it mean "send data into"?  Or "set data to"?


Add leading spaces, as special syntax?

---

```suggestion
  The following example sends data to script for one-time, read-only consumption.
```